core/controllers/admin.py

```diff
@@ -752,6 +912,28 @@ class AddContributionRightsHandler(base.BaseHandler):
     """Handles adding contribution rights for contributor dashboard page."""
 
     GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+    URL_PATH_ARGS_SCHEMAS = {}
+    HANDLER_ARGS_SCHEMAS = {
+        'POST': {
+            'username': {
+                'type': 'string'
+            },
+            'category': {
+                'type': 'string',
+                'choices': [
+                    'translation', 'voiceover',
```

Comment:
Please use predefined constants instead of hardcoding it.

Reply:
Done